common/models/order.json

```diff
@@ -36,6 +36,10 @@
         "columnName": "total",
         "dataType": "real"
       }
+    },
+    "charitableDonation": {
```

Comment:
This needs the PostgreSQL type set so that the number is saved as `dataType: 'real'` rather than the default integer value. This will currently throw an error (or truncate the donation, can't remember the default behaviour) if the user enters a float.